source/blog/2018-05-30-emberjs-accessibility-and-a11y-tools.md

```diff
@@ -0,0 +1,178 @@
+---
+title: "Ember.js accessibility and a11y tools"
+author: Jen Weber
+tags: Recent Posts, a11y, accessibility, testing, addons
+alias : "blog/2018/05/30/ember-accessibility-and-a11y-tools.html"
+responsive: true
+---
+
+With just a few lines of code, you can audit your Ember.js app for accessibility, get customized advice on how fix problems, and see visual indications of which things on a page need work. We'll walk through an example using the [Super Rentals](https://github.com/ember-learn/super-rentals) app from Ember's [official tutorials](https://guides.emberjs.com/current/tutorial/ember-cli/). I will also share some of the latest work being done on Ember.js itself to make it better for accessibility out of the box.
+
+An [accessible app](https://en.wikipedia.org/wiki/Web_accessibility) is one that gives all users equal access to information and functionality, including those who use Assistive Technology like screen readers. This kind of work is sometimes abbreviated as `a11y`. There are set standards called the [Web Content Accessibility Guidelines](https://www.w3.org/WAI/standards-guidelines/wcag/), and in many cases, [it's the law](https://www.w3.org/WAI/Policy/).
+
+_This article was written as of Ember.js version 3.1. The same approach applies for all 2.x apps, with some changes to test syntax. Anyone can write a guest article --- get in touch if you are interested!_
+
+## Using ember-a11y-testing
+
+We don't have to learn a thousand rules to start building accessible apps. There are amazing community contributors who are working hard to make it as easy as possible to follow web best practices, and they created a set of tools referred to as [ember-a11y](https://github.com/ember-a11y).
+
+[ember-a11y-testing](https://github.com/ember-a11y/ember-a11y-testing) is an addon that reveals actionable improvements to your app's accessibility and guides us through making the changes.
+
+Let's start by cloning an example app to try the tools on:
+
+```bash
+git clone git@github.com:ember-learn/super-rentals.git
+cd super-rentals
+yarn # or npm install
+
+Next, install ember-a11y-testing and start the app:
```

Comment:
We need another code fence above here

![](https://dha4w82d62smt.cloudfront.net/items/2f291i2C312o271f051e/Image%202018-05-24%20at%209.05.57%20AM.png?X-CloudApp-Visitor-Id=e3c8ea2634b979f5b9724e227e7e3f95&v=16e73751)

Reply:
Thanks! done